editor/modes/visual-editor/block-list.js

```diff
@@ -106,9 +101,8 @@ export default connect(
 		selectionEnd: getBlockSelectionEnd( state ),
 	} ),
 	( dispatch ) => ( {
-		clearSelectedBlock: () => dispatch( { type: 'CLEAR_SELECTED_BLOCK' } ),
 		onMultiSelect( { start, end } ) {
 			dispatch( { type: 'MULTI_SELECT', start, end } );
 		},
 	} )
-)( clickOutside( VisualEditorBlockList ) );
```

Comment:
Somehow missed this. I'm not sure if this change is a good idea, because now the selection will stay if you click in an input (e.g.) the title, or move the focus elsewhere. This causes backspace to no longer work, and the question is how it should work if the selection is in two places at once. To me is seems better that clicking or moving focus anywhere outside the UI that has to do with multi-selection should deselect.